frontend/public/components/_nav.scss

```diff
@@ -52,39 +53,39 @@ $active-border: 3px;
   -webkit-overflow-scrolling: touch;
 }
 
+.navigation-container--open {
+  .navigation-container__list {
```

Comment:
BEM suggests the avoidance of nesting.  The modifier should go on the child, not the parent.  Here and below.

Nit:  alphabetize rules by selector.  Here and below.